### ISSUE TITLE
Stack trace in response

### ISSUE THREAD

**Issue by TheCallSign** [panmnesia/panmnesia-server#2]
A stack trace is part of the response when a user tries to register with a username already registered.
eg:
```
Error: User already exists
    at Promise.all.then (C:\Users\...\backend\node_modules\panmnesia-server\src\AuthenticationController.js:135:17)
    at <anonymous>
```

Perhaps respond with a JSON error type object like `{"success":false}` because when the user is successfully register `{"success":true}` is the response.

**Comment by TheCallSign** [comment]
I realized now that when `process.env.NODE_ENV === 'production'` the stack trace is removed. However I still think the error response should be JSON.

**Comment by EternalDeiwos** [comment]
I disagree with just a boolean `success` in JSON response in the error case; the status codes should be sufficient for most cases and then maybe one would include an error message:
```javascript
{ "error": "User already exists" }
```
From a security perspective it might not be a good idea to expose too much information in the production environment, but at what point is the information "too much"?

**Comment by TheCallSign** [comment]
Fixed in #5
